translate/canon/DetectLanguage.cs

```diff
@@ -0,0 +1,32 @@
+// Copyright(c) 2017 Google Inc.
+//
+// Licensed under the Apache License, Version 2.0 (the "License"); you may not
+// use this file except in compliance with the License. You may obtain a copy of
+// the License at
+//
+// http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS, WITHOUT
+// WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied. See the
+// License for the specific language governing permissions and limitations under
+// the License.
+//
+// [START translate_detect_language]
+
+using Google.Cloud.Translation.V2;
+using System;
+
+public partial class TranslateSample
```

Comment:
since the class is part of the sample in your proposal, I suggest we either change it to **sealed** or remove the "**partial**" from it.

Reply:
OK, I understand why you did partial. So every sample should be in a separate file so you can use region tags easier?

Reply:
My goal was to get the `using` statements into the sample.  Lots of small files comprising a partial class was the method I chose to accomplish that goal.

Reply:
ok, makes sense.